src/Config/ConfigPanel.ts

```diff
@@ -88,10 +94,33 @@ export class ConfigPanel {
   }
 
   private _getHtmlForWebview(webview: vscode.Webview, context: vscode.ExtensionContext) {
-    // TODO Use a nonce to only allow specific scripts to be run
-    // TDOO fill with logics related to html
-    // TODO import html and change html source to fit webview format
-
-    return '';
+    const scriptUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'index.js'));
+
+    // Uri to load styles into webview
+    const stylesResetUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'reset.css'));
+    const stylesMainUri =
+        webview.asWebviewUri(vscode.Uri.joinPath(this._extensionUri, 'media/Config', 'vscode.css'));
+
+    // Use a nonce to only allow specific scripts to be run
+    const nonce = getNonce();
+
+    // Get html file for webview
+    const filePath: vscode.Uri =
+        vscode.Uri.file(path.join(context.extensionPath, 'media', 'Config', 'index.html'));
+    let html = fs.readFileSync(filePath.fsPath, 'utf8');
+    let re = /\${webview.cspSource}/gi;
+    html = html.replace(re, `${webview.cspSource}`);
+    re = /\${stylesResetUri}/gi;
```

Comment:
Can this be changed to a function? if possible, there are 4 similar patterns that can be made easy to read.

Reply:
Please take this as optional, we can refactor this later.